frontend/apps/filemanager/filemanager.lua

```diff
@@ -1044,14 +1044,25 @@ function FileManager:showFiles(path, focused_file)
     self.instance = file_manager
 end
 
---[[
-A shortcut to execute mv command (self.mv_bin) with from and to as parameters.
-Returns a boolean value to indicate the result of mv command.
---]]
+
+-- A shortcut to execute mv command (self.mv_bin) with from and to as parameters.
```

Comment:
It's not a normal comment, so there are a few special conventions. Otherwise you don't get those nice auto-generated docs like at http://koreader.rocks/doc/modules/util.html#lastIndexOf Three dashes is the activation method.
```suggestion
--- A shortcut to execute mv command.
```

If you want to add a comment about from and to, use
```
-- @param from description goes here
```

But just saying "with from and to as parameters" is redundant and distracting.